app_basic/template.yml

```diff
@@ -0,0 +1,9 @@
+minimum_cli_version: "2.8.0"
+files_to_render:
+  - snowflake.yml
+  - README.md
+variables:
+  - name: project_name
+    prompt: "Project identifier"
+    default: my_native_app_project
```

Comment:
Is there any strong opinion on using "project" for this variable?

Reply:
`snow init` still says it "creates a project directory", so I figured that was the right term.